docs/CRI/containerd.md

```diff
@@ -114,6 +114,15 @@ containerd_runc_runtime:
   ...
 ```
 
+The `containerd_extra_runtime_args` variable allows you to inject additional configuration options into the containerd CRI runtime plugin section (`[plugins."io.containerd.cri.v1.runtime"]`).
+
+This is useful for adding containerd runtime configuration options that aren't explicitly supported by Kubespray's default variables.
+
+```yaml
+containerd_extra_runtime_args: |
+  device_ownership_from_security_context = true
```

Comment:
Two things:
- this should instead be in the role defaults with the docs as a comment
- It's probably better to have this as a dict and use a for in the jinja template (with {{ key }} = {{ value }}).
Structured data makes it easier to make the templates evolve / merge stuff, etc.